src/hooks/AuthHooks.res

```diff
@@ -20,6 +20,11 @@ let getHeaders = (~uri, ~headers, ()) => {
       "accept": "application/json",
     }
     Fetch.HeadersInit.make(headerObj)
+  } else if isFromFormData {
+    let headerObj = {
+      "Authorization": `Bearer ${hyperSwitchToken->Belt.Option.getWithDefault("")}`,
+    }
+    Fetch.HeadersInit.make(headerObj)
```

Comment:
the HeadersInit.make function is being called multipleTimes, can this be reduced?

Reply:
It was not possible to move the `Fetch.HeadersInit.make` at the end as it takes a record type.

Reply:
It's done in recent changes.